lib/query-conversation/src/main/java/io/redlink/smarti/query/conversation/ConversationQueryBuilder.java

```diff
@@ -152,6 +148,16 @@ protected final void addClientFilter(final SolrQuery solrQuery, Conversation con
         solrQuery.addFilterQuery(new StringBuilder(FIELD_OWNER).append(':')
                 .append(conversation.getOwner().toHexString()).toString());
     }
-
+    /**
+     * Adds a FilterQuery for the passed supportArea
+     * @param solrQuery
+     * @param supportArea
+     */
+    protected final void addSupportAreaFilter(final SolrQuery solrQuery, final String supportArea){
+        if(StringUtils.isNoneBlank(supportArea)){
+            solrQuery.addFilterQuery(new StringBuilder(getEnvironmentField(Context.ENV_SUPPORT_AREA)).append(':')
+                    .append(supportArea).toString());
```

Comment:
If `supportArea` is a String value that is simply appended to the `ENV_SUPPORT_AREA` param, we do not need a multi valued field, do we? See my comment below.

Reply:
I know, 'supportArea' is single valued but their might be other `env_*` fields that are multi valued. Because of that I changed the `<dynamicField name="env_*" .. \>` to be multi valued.